network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -160,7 +160,7 @@ class BlogPage(BasePage):
         help_text="Check this box to add a comment section for this blog post.",
     )
 
-    related_post_count = 3
+    RELATED_POSTS_MAX = 3
```

Comment:
Renamed for clarity on what it means. 

`related_post_count` was just too close to `related_posts.count()`.

Reply:
Makes sense. Thanks!